configs/9.0/packages/expat/sources

```diff
@@ -40,12 +42,45 @@ ATSRC_PACKAGE_BUNDLE=main_toolchain
 
 atsrc_get_patches ()
 {
-        at_get_patch \
-	https://hg.mozilla.org/releases/mozilla-esr31/raw-diff/2f3e78643f5c/parser/expat/lib/xmlparse.c \
-	978566c4144504d170869619661fbd7d || return ${?}
```

Comment:
OK.  This was incorporated into expat by commit ID ba0f9c3b40c2 (between 2.1.0 and 2.1.1).